enterprise/frontend/src/embedding-sdk/components/public/MetabotQuestion/QuestionDetails.tsx

```diff
@@ -0,0 +1,82 @@
+import { t } from "ttag";
+
+import { useBreakoutData } from "embedding-sdk/components/private/InteractiveQuestion/components/Breakout/use-breakout-data";
+import { useFilterData } from "embedding-sdk/components/private/InteractiveQuestion/components/Filter/hooks/use-filter-data";
+import { useSummarizeData } from "embedding-sdk/components/private/InteractiveQuestion/components/Summarize/use-summarize-data";
+import { useInteractiveQuestionContext } from "embedding-sdk/components/private/InteractiveQuestion/context";
+import { isNotNull } from "metabase/lib/types";
+import { Text } from "metabase/ui";
+import * as Lib from "metabase-lib";
+import type Question from "metabase-lib/v1/Question";
+
+export function QuestionDetails() {
```

Comment:
let me know if you have any suggestion simplifying this.

Reply:
I think this is already very readable - it's clear what it is doing 👍🏻